install.sh

```diff
@@ -5,19 +5,19 @@ minorversion=$(gedit --version | sed -n 's/gedit.*\s[0-9]\.\([0-9]*\).*/\1/p')
 installfiles="editorconfig_plugin/"
 editorconfigcore="editorconfig-core-py/"
 
-if [ "$majorversion" -eq "3" ] ; then
-    localinstalldir=~/.local/share/gedit/plugins
-    rootinstalldir=/usr/lib/gedit/plugins
-    installfiles="$installfiles $editorconfigcore editorconfig_gedit3.py editorconfig.plugin"
-else
+localinstalldir=~/.local/share/gedit/plugins
+rootinstalldir=/usr/lib/gedit/plugins
+installfiles="$installfiles $editorconfigcore editorconfig_gedit3.py editorconfig.plugin"
+
+if [ "$majorversion" -lt "3" ] ; then
     localinstalldir=~/.gnome2/gedit/plugins
     rootinstalldir=/usr/lib/gedit-2/plugins
     installfiles="$installfiles $editorconfigcore editorconfig_gedit2.py editorconfig.gedit-plugin"
 fi
 
-if [ "$(id -u)" -ne "0" ] ; then
-    installdir=$localinstalldir
-else
+installdir=$localinstalldir
+
+if [ "$(id -u)" -eq "0" ] ; then
```

Comment:
Do you have a reason to make changes here? The logic of the version before change looks clearer to me.

Reply:
I believe I was following the logic of the previous chunk (assume gedit 3 as the default, then test for gedit 2). I would presume that most users would install the plugin for themselves only, with installing system-wide as a special case.

Can change this back to the previous version if you prefer.